src/dev/code_coverage/ingest_coverage/integration_tests/mocks/jest-combined/bulk-data.json

```diff
@@ -1897,7 +1897,7 @@
       "pct": 100
     }
   },
-  "/var/lib/jenkins/workspace/elastic+kibana+code-coverage/kibana/src/legacy/core_plugins/data/public/actions/filters/create_filters_from_event.js": {
+  "/var/lib/jenkins/workspace/elastic+kibana+code-coverage/kibana/src/legacy/core_platform/plugins/shared/data/public/actions/filters/create_filters_from_event.js": {
```

Comment:
This is just a mock so probably not a big deal, but it looks like there may have been a find/replace issue here.